server/views/resources/code-of-conduct.jade

```diff
@@ -3,23 +3,18 @@ block content
-            | All campers are required to agree to and follow this code of conduct.
-        p 
-            | In short: Be nice. No harassment, Trolling, or Spamming.
+        p Free Code Camp is a friendly place to learn to code. We’re committed to keeping it that way.
+        p By using Free Code Camp, you agree that you'll follow this code of conduct.
+        p In short: <bold>Be nice. No harassment, trolling, or spamming.</bold>
         ul
-            li <strong>Harassment</strong> includes sexual language and imagery, deliberate intimidation, stalking, unwelcome sexual attention, libel and any malicious hacking or social engineering. Free Code Camp should be a harassment-free experience for everyone,regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, national origin, or religion (or lack thereof).
-            li <strong>Trolling</strong> includes posting inflamitory comments to provoke an emotional response or disrupt discussions.
-            li <strong>Spamming</strong> includes posting off-topic messages to disrupt discussions or promote a product.
-        p
-            | If you see someone harass, troll, or spam anywhere on Free Code Camp's online media (Gitter, YouTube, Facebook, etc.), notify us in our &thinsp;
+            li <bold>Harassment</bold> includes sexual language and imagery, deliberate intimidation, stalking, unwelcome sexual attention, libel and any malicious hacking or social engineering. Free Code Camp should be a harassment-free experience for everyone, regardless of gender, gender identity and expression, age, sexual orientation, disability, physical appearance, body size, race, national origin, or religion (or lack thereof).
+            li <bold>Trolling</bold> includes posting inflamitory comments to provoke an emotional response or disrupt discussions.
+            li <bold>Spamming</bold> includes posting off-topic messages to disrupt discussions or promote a product.
+        p If you see someone harass, troll, or spam anywhere on Free Code Camp's online media (Gitter, YouTube, Facebook, etc.), notify us in our &thinsp;
             a(href='https://gitter.im/freecodecamp/admin' target='_blank') Admin room in Gitter
             | &thinsp; - preferably with a screen shot of the offense. Our team will take any action we deem appropriate, up to and including banning the offender from Free Code Camp.
-        p
-            | Also, no bots are allowed in any of our Official Chat Rooms without prior written permission from the &thinsp;
+        p Also, no bots are allowed in our chat rooms or on our forum without prior written permission from &thinsp;
             a(href='https://gitter.im/FreeCodeCamp/CoreTeam' target="_blank") Free Code Camp's Core Team.
-        p
-            | If you have questions about this code of conduct, email us at&thinsp;
+        p If you have questions about this code of conduct, email us at&thinsp;
```

Comment:
We should use "this" not our here, because we want to make sure people realize that this is our one single code of conduct.